packages/make-styles-webpack-loader/config/tests.js

```diff
@@ -1 +1,4 @@
 /** Jest test setup file. */
+
+// Tests are using webpack to compile code, default timeout could be not enough to pass tests
+jest.setTimeout(30000);
```

Comment:
hmm this will drastically slow down our pipeline. isn't there a better way? is this well known behaviour ?

Reply:
> Note: The default timeout interval is 5 seconds if this method is not called.
> Note: If you want to set the timeout for all test files, a good place to do this is in setupFilesAfterEnv.

https://jestjs.io/docs/jest-object#jestsettimeouttimeout

---

- 5 seconds is not enough to pass the first test with Webpack on CI (others are passing) and it fails on CI
- We can't mock Webpack and we want to test integration (and it's recommended in Webpack docs)

Do you have suggestions?

Reply:
> Do you have suggestions?

I don't that's why I asked :) 

looks like it is what it is. thx!